app/controllers/api/invitations_controller.rb

```diff
@@ -19,12 +19,14 @@ def remove
 
   def update
     raw_invitation_token = update_resource_params["invitation_token"]
-    self.resource = accept_resource
+    user = User.find_by_invitation_token(raw_invitation_token, true)
```

Comment:
This seems a little weird. What sort of errors are we getting?

Reply:
What happens here is that because the user is accepted and obtained at the same time, if the user does not already have a first and last name it will fail validation. The user must have its first and last name set before confirmation.